app/src/models/popup.ts

```diff
@@ -102,7 +103,11 @@ export type Popup =
       repository: Repository
       branch?: Branch
     }
-  | { type: PopupType.RepositorySettings; repository: Repository }
+  | {
+      type: PopupType.RepositorySettings
+      repository: Repository
+      initialSelectedTab?: RepositorySettingsTab
```

Comment:
This is used to allow opening the repository settings dialog from a specific tab.